forecasting/src/autogluon/forecasting/models/gluonts/models.py

```diff
@@ -0,0 +1,259 @@
+    Estimators). While this class is meant to generally enable fast use of GluonTS
+    models in autogluon, specific GluonTS models accessed through this wrapper may
+    not have been tested and should be used at the user's own risk.
+
+    Please refer to each GluonTS estimator's individual documentation for
+    initialization parameters of each model.
+
+    Parameters
+    ----------
+    gluonts_estimator_class:
+        The class object of the GluonTS estimator to be used.
+    """
+
+    def __init__(self, gluonts_estimator_class: Type[GluonTSEstimator], **kwargs):
+        self.gluonts_estimator_class = gluonts_estimator_class
+        gluonts_model_name = re.sub(
+            r"Estimator$", "", self.gluonts_estimator_class.__name__
+        )
+
+        super().__init__(name=kwargs.pop("name", gluonts_model_name), **kwargs)
+
+    def get_params(self) -> dict:
+        params_dict = super().get_params()
+        params_dict["gluonts_estimator_class"] = self.gluonts_estimator_class
+        return params_dict
+
+
+class _ProphetDummyEstimator(DummyEstimator):
+    def train(self, train_data, validation_data=None, **kwargs):
+        return self.predictor
+
+
+class ProphetModel(AbstractGluonTSModel):
+    """Wrapper around `Prophet <https://github.com/facebook/prophet>`_, which wraps the
+    library through GluonTS's wrapper.
+
+    In order to use it you need to install the package::
+
+        # you can either install Prophet directly
+        pip install fbprophet
+
+        # or install gluonts with the Prophet extras
+        pip install gluonts[Prophet]
+
+    Other Parameters
+    ----------------
+    hyperparameters
+        Model hyperparameters that will be passed directly to the `Prophet`
+        class. See Prophet documentation for available parameters.
+    """
+
+    gluonts_estimator_class = _ProphetDummyEstimator
+    allowed_prophet_parameters = [
+        "growth",
+        "changepoints",
+        "n_changepoints",
+        "changepoint_range",
+        "yearly_seasonality",
+        "weekly_seasonality",
+        "daily_seasonality",
+        "holidays",
+        "seasonality_mode",
+        "seasonality_prior_scale",
+        "holidays_prior_scale",
+        "changepoint_prior_scale",
+        "mcmc_samples",
+        "interval_width",
+        "uncertainty_samples",
+    ]
+
+    def _get_estimator(self) -> GluonTSEstimator:
+        model_init_params = self._get_estimator_init_args()
+
+        return _ProphetDummyEstimator(
+            predictor_cls=ProphetPredictor,
+            freq=model_init_params["freq"],
+            prediction_length=model_init_params["prediction_length"],
+            prophet_params={
+                k: v
+                for k, v in model_init_params.items()
+                if k in self.allowed_prophet_parameters
+            },
+        )
+
+
+# TODO: AutoGluon Tabular will be removed from GluonTS to avoid circular dependencies
```

Comment:
Another (temporary) hack may be if GluonTS just depends on autogluon.tabular sub-package instead of the full  autogluon.
Long term though, the AutoTabularModel should probably be moved back into just AG as it's not a core part of GluonTS.

Reply:
GluonTS does not have autogluon as a core dependency, but only as an optional requirement for users who want to use AG Tabular; so I think we are safe in the near future.